detection_tables.h

```diff
@@ -198,6 +198,34 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	{
 		{
+			"melements",
```

Comment:
Please add here comments which those versions are, and that you have them. Like here:

https://github.com/scummvm/scummvm/blob/master/engines/sci/detection_tables.h#L176